blueocean/pom.xml

```diff
@@ -15,13 +15,6 @@
     <url>https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean/doc/BlueOcean.adoc</url>
     <description>Blue Ocean is a new project that rethinks the user experience of Jenkins. Designed from the ground up for Jenkins Pipeline and compatible with Freestyle jobs, Blue Ocean reduces clutter and increases clarity for every member of your team.</description>
 
-    <scm>
```

Comment:
why this? is it related to this PR?

Reply:
https://github.com/jenkinsci/blueocean-plugin/pull/2385#discussion_r1081778593